core/tx_pool.go

```diff
@@ -55,6 +56,15 @@ var (
 	evictionInterval     = time.Minute   // Time interval to check for evictable transactions
 )
 
+var (
+	// Metrics
+	pendingDiscardMeter    = metrics.NewMeter("core/txpool/transactions/pending/discard")
+	pendingReplaceMeter    = metrics.NewMeter("core/txpool/transactions/pending/replace")
+	queuedDiscardMeter     = metrics.NewMeter("core/txpool/transactions/queued/discard")
+	queuedReplaceMeter     = metrics.NewMeter("core/txpool/transactions/queued/replace")
+	invalidTxMeter         = metrics.NewMeter("core/txpool/transactions/invalid")
```

Comment:
Done.  But PTAL that I added the new drop-counters at the right places, the drop-logic is a bit peculiar....